.Rbuildignore

```diff
@@ -5,3 +5,6 @@
 cran-comments.md
 tests/testthat/test-integration_tests.R
 inst/testdata/shakespeare_mapping.json
+CONDUCT.md
+LICENSE.MD
+.travis.yml
```

Comment:
I'd suggest adding
- all the stuff in the .gitignore
- .gitignore itself
- ^ and $ and \ to match regex better

Reply:
good call! will add them right now

Reply:
added!